### ISSUE TITLE
Various input fields become unreadable when using a dark theme in OS that changes the default input styles in your browser

### ISSUE THREAD

**Issue by ZebulanStanphill** [front/drupal-gutenberg#5]
## The issue
I am using the Breeze Dark theme for KDE Plasma on Linux. This changes the default styling for input fields in my browser, Firefox. When something overrides the text color but not the background color, the inputs end up appearing nearly unreadable:
![image](https://user-images.githubusercontent.com/19592990/45111170-e46d7200-b109-11e8-9bd9-c57ac7880b59.png)

A simple solution would be to set a background color for all the input fields to override whatever defaults are set.

Tested via the demo here: https://drupalgutenberg.org/demo

**Comment by perandre** [comment]
Do you have the same issue on https://testgutenberg.com/ @ZebulanStanphill ? If yes, this should be reported up stream to https://github.com/WordPress/gutenberg/

**Comment by ZebulanStanphill** [comment]
@perandre No, the input fields have background colors set there that override the browser/OS defaults.

**Comment by perandre** [comment]
Thanks for the information – lining it up to be fixed after Drupal Europe.
